### DISCUSSION TITLE
Daire içindeki kare mi , kare içindeki daire mi sorusu

### DISCUSSION THREAD

**Discussion by sonsuzus** [sonsuzus/sonsuzus#77] [Zeka Sorusu]
Bir dairenin ve karenin alanı aynı olsun. Daire içine kare, kare içine de daire çizilsin.

Daire içine çizilen karenin alanı mı, kare içine çizilen dairenin alanı mı daha büyük olur?

**Comment by HabilCcabbar** [comment]
hocam pi'ye değer vermeli miyiz?

**Comment by Furkanyildirm** [comment]
pi nin 3 olduğunu ve alanların 300 olduğunu düşünelim karenin içindeki dairenin alanı 3rr formülünden 225 çıkıyor. dairenin içindeki karenin alanını da geogebra(görsel işlem yapma uygulaması) dan yardım alarak 200 buldum. yani dairenin alanı daha büyük

**Comment by begumasyaburus** [comment]
Karenin bir kenarı a dairenin yarıçapı r olsun. 
Karenin içerisine daire çizdiğimizde dairenin yarı çapı r iken karenin bir kenar uzunluğu 2r olacağından yani a=2r olduğundan alan formüllerine baktığımızda, pi=4 olarak alınmadığı sürece eşitlik söz konusu olamaz. Pi 4'ten küçük olduğu için "pi r kare küçüktür a kare" eşitsizliği oluşur.
Bu durumda, dairenin içerisine çizilen karenin alanı dairenin alanından büyüktür. (Bu soruyu karenin içerisine daire çizerek çözdüğümde bu şekilde bir sonuç buldum)
